### DISCUSSION TITLE
Can't access to my workspace of CS50 in codespace

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27408] [Codespaces]
A few hours ago I was working in my workspace when my laptop shutdown. I wait like 5 hours to access again but now codespace can't access to my workspace. It just keep Setting up remote connection and do nothing. Someone can help me?
![mario c - 109868484 - Visual Studio Code](https://user-images.githubusercontent.com/109868484/181412590-46d4dd89-e62a-42c7-b0b7-d6058e865dac.jpg)

**Comment by xRodzMC** [comment]
Please check your connection first. If problem persist try to view this [article about codespace troubleshooting](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-codespaces-clients)

> **Note**
> If you found this helpful, Please make as answer

**Reply by ghost** [reply]
Thank you, I will try that.

**Reply by xRodzMC** [reply]
No problem

**Comment by jfullstackdev** [comment]
slow connection, low memory, GitHub extensions, you may want to consider these things too

**Comment by jungaretti** [comment]
There was a regional outage affecting Codespaces last night: https://www.githubstatus.com/incidents/7rghmvv5crq0

You should be able to reconnect today!
